docs/operator-docs/lifecycle-policies.md

```diff
@@ -0,0 +1,152 @@
+---
+id: lifecycle-policies
+title: Lifecycle Policies
+hide_title: true
+---
+
+# Setting Lifecycle Policies
```

Comment:
Should we call this section setting S3 Lifecycle Policies since it's S3 specific?

Reply:
Yes, will make changes